sdk/python/foundation-models/system/inference/zero-shot-image-classification/zero-shot-image-classification-online-endpoint.ipynb

```diff
@@ -0,0 +1,384 @@
+    "timestamp = int(time.time())\n",
+    "online_endpoint_name = \"zero-shot-image-cls-\" + str(timestamp)\n",
+    "# Create an online endpoint\n",
+    "endpoint = ManagedOnlineEndpoint(\n",
+    "    name=online_endpoint_name,\n",
+    "    description=\"Online endpoint for \"\n",
+    "    + foundation_model.name\n",
+    "    + \", for zero-shot-image-classification task\",\n",
+    "    auth_mode=\"key\",\n",
+    ")\n",
+    "workspace_ml_client.begin_create_or_update(endpoint).wait()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import OnlineRequestSettings, ProbeSettings\n",
+    "\n",
+    "deployment_name = \"zero-shot-image-mlflow-deploy\"\n",
+    "\n",
+    "print(foundation_model.id)\n",
+    "print(online_endpoint_name)\n",
+    "print(deployment_name)\n",
+    "\n",
+    "# Create a deployment\n",
+    "demo_deployment = ManagedOnlineDeployment(\n",
+    "    name=deployment_name,\n",
+    "    endpoint_name=online_endpoint_name,\n",
+    "    model=foundation_model.id,\n",
+    "    instance_type=\"Standard_DS3_V2\",  # Use GPU instance type like Standard_NC6s_v3 for faster inference\n",
+    "    instance_count=1,\n",
+    "    request_settings=OnlineRequestSettings(\n",
+    "        max_concurrent_requests_per_instance=1,\n",
+    "        request_timeout_ms=90000,\n",
+    "        max_queue_wait_ms=500,\n",
+    "    ),\n",
+    "    liveness_probe=ProbeSettings(\n",
+    "        failure_threshold=49,\n",
+    "        success_threshold=1,\n",
+    "        timeout=299,\n",
+    "        period=180,\n",
+    "        initial_delay=180,\n",
+    "    ),\n",
+    "    readiness_probe=ProbeSettings(\n",
+    "        failure_threshold=10,\n",
+    "        success_threshold=1,\n",
+    "        timeout=10,\n",
+    "        period=10,\n",
+    "        initial_delay=10,\n",
+    "    ),\n",
+    ")\n",
+    "workspace_ml_client.online_deployments.begin_create_or_update(demo_deployment).wait()\n",
+    "endpoint.traffic = {deployment_name: 100}\n",
+    "workspace_ml_client.begin_create_or_update(endpoint).result()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5. Test the endpoint\n",
+    "\n",
+    "We will fetch some sample data from the test dataset and submit to online endpoint for inference."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "demo_deployment = workspace_ml_client.online_deployments.get(\n",
+    "    name=deployment_name,\n",
+    "    endpoint_name=online_endpoint_name,\n",
+    ")\n",
+    "\n",
+    "# Get the details for online endpoint\n",
+    "endpoint = workspace_ml_client.online_endpoints.get(name=online_endpoint_name)\n",
+    "\n",
+    "# Existing traffic details\n",
```

Comment:
We can remove this as we initialized to 100% just above. Even I removed it from stablediffusion notebook